src/error_functions.py

```diff
@@ -5,18 +5,34 @@
 def mean_euclidean_error(predicted, target):
     """
     Computes the Mean Euclidean Error between
-    the targ vector and the output pred by the net
+    the targ vector and the output pred by the net over all patterns
 
     :param predicted: ndarray of shape (n, m) – Predictions for the n examples
     :param target: ndarray of shape (n, m) – Ground truth w_vals for each of n examples
     :return: loss in term of mee (Mean Euclidean Error)
     """
+
     if predicted.shape != target.shape:
-        raise Exception(f"Mismatching shapes in MSE: predictions shape: "
+        raise Exception(f"Mismatching shapes in MEE: predictions shape: "
                         f"{predicted.shape} - targets shape {target.shape}")
     return np.linalg.norm(predicted - target) / target.shape[0]
 
 
+def mean_squared_error(predicted, target):
+    """
+    Computes the Mean Squared Error between
+    the targ vector and the output pred by the net over all patterns
+
+    :param predicted: ndarray of shape (n, m) – Predictions for the n examples
+    :param target: ndarray of shape (n, m) – Ground truth w_vals for each of n examples
+    :return: loss in term of mse (Mean Squared Error)
+    """
+    if predicted.shape != target.shape:
+        raise Exception(f"Mismatching shapes in MSE: predictions shape: "
```

Comment:
Anche questo cambiamolo in `AttributeError`,
poi occhio al corrispondente unit test